src/fastfield/serializer/mod.rs

```diff
@@ -79,6 +79,11 @@ impl CompositeFastFieldSerializer {
         self.composite_write.for_field_with_idx(field, 1)
     }
 
+    /// Gets the underlying writer
+    pub fn get_field_writer(&mut self, field: Field, idx: usize) -> &mut impl Write {
+        self.composite_write.for_field_with_idx(field, idx)
```

Comment:
I think the other fast field writer leaves the serialization logic in the serializer.

THe fast field writer accumulates values in memory, and then is in charge of passing a view over the data to the serializer which in turn serializes. 

If this is correct, why did you change the logic?

Reply:
It's not true for the bytes fast field.

I think `CompositeFastFieldSerializer` has evolved over time to something where we can consider removing it. I can pass everything through it, but there's no logic inside it.

Reply:
The bytes fast field is a bit special in that it does not do anything with the bytes.

The idea is that the serializer contains code about the format of the fast field.
That code is common to both the merge and to segment writing.

Please fix